build.ps1

```diff
@@ -443,6 +444,43 @@ function Invoke-PushPackage {
     Invoke-Step { PushPackage }
 }
 
+function CopyTempFilesToDockerContext {
+	$sourceArtifacts = "$solutionRoot/EdFi.Ods.AdminApi/Artifacts/"
+	$destinationArtifacts = "$dockerRoot/"
```

Comment:
@lchavarriaga 
The Artifacts folder is under of "EdFi.Ods.AdminApi". So we can read it from there. Copying EdFi.Ods.AdminApi should be enough